fission/src/ui/modals/spawning/SpawningModals.tsx

```diff
@@ -7,7 +7,8 @@ import { useModalControlContext } from "@/ui/ModalContext"
 import Label, { LabelSize } from "@/components/Label"
 import { CreateMirabufFromUrl } from "@/mirabuf/MirabufSceneObject"
 import World from "@/systems/World"
-import { useTooltipControlContext } from "@/ui/TooltipContext"
+import { useTooltipControlContext } from "@/TooltipContext"
```

Comment:
```suggestion
import { useTooltipControlContext } from "@/ui/TooltipContext"
```

Sneaky merge conflict.